### PR TITLE
Reorg docs

### PR DESCRIPTION
**Pull request by dhatch**
- Add more
- Remove examples section. Examples => more.
- Add great bash scripts.
- Reorder project info
- Add backlink to old docs
- Link to gitclub app at top level

### CODE REVIEW

File: docs/content/any/project/old-docs.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: 0.15 Documentation
```
Comment (gkaemmer) [inline docs/content/any/project/old-docs.md:2]: Thats neat -- this is so hugo doesn't complain about broken links?

File: docs/scripts/forall [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (gkaemmer) [inline docs/scripts/forall:1]: Probably need usage guide here -- can do in a follow-on

File: docs/themes/oso-tailwind/layouts/partials/sidebar.html [hunk 3]
```diff
@@ -67,6 +67,13 @@
     {{ if eq $level 0 }}font-bold py-2{{else}}font-normal{{end}}
     {{ if $isCurrent}}active text-primary{{else}} text-black{{ end }}
 ">
+    {{ if isset .Params "href" }}
```
Comment (gkaemmer) [inline docs/themes/oso-tailwind/layouts/partials/sidebar.html:70]: What is this for?

**Review by gkaemmer — COMMENTED** [review]
I think this looks good -- can approve if you think it's ready

File: docs/content/any/project/old-docs.md (hunk 1 above)
Comment (dhatch) [reply docs/content/any/project/old-docs.md:2]: for this somehow yeah

File: docs/themes/oso-tailwind/layouts/partials/sidebar.html (hunk 3 above)
Comment (dhatch) [reply docs/themes/oso-tailwind/layouts/partials/sidebar.html:70]: to make the gitclub link work

**Review by gkaemmer — APPROVED** [review]

**Pull request merged**